scalableminds/webknossos
# Fix welcome notification on wk.org

This PR is a follow up for the antd v5 upgrade and fixes the styling of the "Welcome"/"Sign up for free" notification on wk.org. 

**Before**:
<img width="476" alt="image" src="https://github.com/scalableminds/webknossos/assets/1105056/abac8bd5-14fb-4ff9-8aba-9f03a994ba7c">


**After**:
<img width="514" alt="image" src="https://github.com/scalableminds/webknossos/assets/1105056/a5cd24b8-7b7e-46ca-9718-330a88eebc44">


### Steps to test:
- Copy the URI/link to a dataset for them the toolbar
- Open the link in an incognito browser window
- The welcome notification should show


### Issues:
- follow up to #6861 

------
(Please delete unneeded items, merge only when none are left open)
